beniget/beniget.py

```diff
@@ -290,14 +468,61 @@ class DefUseChains(ast.NodeVisitor):
     One instance of DefUseChains is only suitable to analyse one AST Module in it's lifecycle.
     """
 
-    def __init__(self, filename=None):
+    def __init__(self,
```

Comment:
I'm stopping my review here, I think we discussed you should try to subclass beniget instead of adding a conditional path in it. Did you change your mind?